.azure-pipelines/steps/download-artifact.yml

```
@@ -0,0 +1,21 @@
+# .azure-pipelines/steps/download-artifact.yml
+parameters:
+  artifact: ''
+  path: ''
+  patterns: ''
+  displayName: ''
+  condition: ''
+  # Defaults
+  timeoutInMinutes: 45
```

I think we should reduce this a _lot_. Download should probably never take more than 3 minutes, so maybe 10 is a safer bet here?
```suggestion
  timeoutInMinutes: 10
```